docs/v4/objects/request.md

```diff
@@ -295,6 +295,27 @@ methods to read and iterate its underlying PHP `resource`.
 * getContents()
 * getMetadata($key = null)
 
+### Testing the Request Body
+
+When testing POST actions, you will need to set a body on the request. The `TestCase` helper included with the Slim skeleton features a `->createRequest` method that accepts a `$method` and a `$path`. It creates an internal resource pointing to `php://temp`, which you might be able to write to.
```

Comment:
I wanted to end this paragraph on a stronger note, but ... it didn't seem to be right to call file_put_contents in a test, nor to fopen `php://temp` when it was already open elsewhere as a separate resource (would it open two separate temp streams? I dunno.) Maybe the whole bit about `php://temp` should be removed?